rust/src/main.rs

```diff
@@ -9,134 +9,38 @@ async fn async_main() {
-        Err(e) => println!("{:?}", e),
-        _ => ()
-    }
-    match private::opening_new_order_async(instrument, 1, Some(35000.to_string()), 2, "0.0001").await {
-        Err(e) => println!("{:?}", e),
-        _ => ()
-    }
-    match private::opening_new_order_async(instrument, 1, Some("".to_string()), 1, "0.0001").await {
-        Err(e) => println!("{:?}", e),
-        _ => ()
-    }
+    println!("{:?}", public::get_server_time_async().await);
+    println!("{:?}", public::get_markets_async().await);
+    println!("{:?}", public::get_products_async().await);
+    println!("{:?}", public::get_assets_async().await);
+    println!("{:?}", public::get_crypto_assets_async().await);
+    println!("{:?}", public::get_fiat_assets_async().await);
+    println!("{:?}", public::get_asset_pairs_async("crypto","spot","ID").await);
+    println!("{:?}", public::get_ticker_async(instrument).await);
+    println!("{:?}", public::get_trades_async("2", instrument).await);
+    println!("{:?}", public::get_depth_async(instrument, "5").await);
+    println!("{:?}", public::get_contract_async(instrument_contract).await);
+    println!("{:?}", public::get_insider_profiles_async(None).await);
+    println!("{:?}", public::get_insider_trades_async(None).await);
+    println!("{:?}", public::get_announcements_async().await);
+    println!("{:?}", public::get_order_types_async().await);
+    println!("{:?}", public::get_languages_async().await);
+    println!("{:?}", public::get_search_results_async().await);
+    println!("{:?}", public::get_price_index_async(instrument, None, None, None, None).await);
+    println!("{:?}", public::get_search_async("ID", None).await);
+    println!("{:?}", public::get_matching_engine_order_lag_async(None).await);
+    println!("{:?}", public::get_matching_engine_throughput_async(None).await);
+    println!("{:?}", private::get_account_balance_async().await);
+    println!("{:?}", private::get_account_statement_async().await);
+    println!("{:?}", private::get_transaction_histories_async().await);
+    println!("{:?}", private::get_open_orders_async(instrument, 0, 0).await);
+    println!("{:?}", private::get_closed_orders_async(instrument, 0, 0).await);
+    println!("{:?}", private::get_order_info_async("3f128876-a082-4b18-8fd4-abbcc4aa7915").await);
+    println!("{:?}", private::get_trade_history_async(instrument, 2, 0, 0).await);
+    println!("{:?}", private::cancelling_open_order_per_instrument_async(instrument).await);
+    println!("{:?}", private::cancelling_open_order_per_orderids_async(vec!["5033216a-dad5-4cb6-87d6-0084603be699", "3d19d8d1-c576-4109-8671-5e4115cf6e5e"], vec![-1, -1]).await);
+    println!("{:?}", private::opening_new_order_async(instrument, 1, Some(35000.to_string()), 2, "0.0001").await);
+    println!("{:?}", private::opening_new_order_async(instrument, 1, Some("".to_string()), 1, "0.0001").await);
 }
```

Comment:
you are missing cancel order @agustinustheo

Reply:
i think cancel order is in line 40-41 mas @dendisuhubdy